src/js/window/main-window.js

```diff
@@ -483,9 +486,46 @@ let loadBoardUI = ()=> {
   })
 
   document.querySelector('#open-in-photoshop-button').addEventListener('pointerdown', (e)=>{
+    let children = ['main', 'reference', 'notes'].map(layerName => {
+      return {
+        "name": layerName,
+        "canvas": storyboarderSketchPane.getLayerCanvasByName(layerName)
+      }
+    });
+    let psd = {
+      width: storyboarderSketchPane.canvasSize[0],
+      height: storyboarderSketchPane.canvasSize[1],
+      children: children
+    };
     let board = boardData.boards[currentBoard]
-    let imageFilename = path.join(boardPath, 'images', board.url)
-    shell.openItem(imageFilename)
+    let imageFilePath = path.join(boardPath, 'images', `board-${board.number}.psd`)
+    const buffer = writePsd(psd);
+    fs.writeFileSync(imageFilePath, buffer);
+    shell.openItem(imageFilePath);
+
+    fs.watchFile(imageFilePath, (cur, prev) => {
+      initializeCanvas((width, height) => {
+        let canvas = document.createElement('canvas');
+        canvas.width = width;
+        canvas.height = height;
+        return canvas;
+      });
+      const buffer = fs.readFileSync(imageFilePath);
+      const psd = readPsd(buffer);
+      if(!psd || !psd.children) {
+        return;
+      }
+      let mainCanvas = storyboarderSketchPane.getLayerCanvasByName("main");
+      let ctx = mainCanvas.getContext('2d');
+      for(let layer of psd.children) {
+        if(layer.name == "notes" || layer.name == "reference" || !layer.canvas) {
+          continue;
+        }
+        ctx.drawImage(layer.canvas, layer.left, layer.top);
+      }
+      markImageFileDirty([1]);
```

Comment:
`markImageFileDirty` accepts an array of the layer indexes that have changed. In this case, it looks like all three layers (main, reference and notes) are always drawn, so `markImageFileDirty` should take `[0, 1, 3]`.

The layer indexes are hardcoded currently, kind of a hack. You can see which is which here: https://github.com/wonderunit/storyboarder/blob/master/src/js/window/storyboarder-sketch-pane.js#L73-L80